src/mol-canvas3d/passes/pick.ts

```diff
@@ -138,6 +142,9 @@ export class PickHelper {
 
             this.setupBuffers();
         }
+
+        // TODO: make spiral size configurable
+        this.spiral = spiral2d(Math.ceil(this.pickScale * 5));
```

Comment:
I know the spiral helps for lines, but for bigger objects, e.g. residues in cartoon, it feels a bit weird when the residue lights up when you are 5 pixels from it.

Reply:
Hmm, did not feel odd to me when playing around with it. Pixels are so small nowadays. Anyway, I will make it configurable and am happy to make the default smaller than 5. Also consider that currently the pick buffers are rendered at quarter resolution. So there are already a few pixels +/- off target. Maybe `Math.round(this.pickScale * pickPadding)` with `defaultPickPadding = 3`?

Reply:
Maybe, could be better